export const run = async () => {
  try {
    console.log("hello");
  } catch (error) {
    console.error(error);
  }
};
